docs/_themes/clusterhq/layout_homepage.html

```diff
@@ -117,8 +117,8 @@ <h2 class="heading text-center doc-banner">See our <a href="https://docs.cluster
 				<div><span class="widget-title">Documentation</span>			
 				<div class="textwidget">
 				<ul>
-					<li><a href="#">Getting Started</a></li>
-					<li><a href="https://docs.clusterhq.com/en/latest/gettinginvolved/index.html">Collaborate</a></li>
+					<li><a href="{{ pathto('gettingstarted/index') }}">Getting Started</a></li>
+					<li><a href="{{ pathto('gettinginvolved/contributing') }}">Collaborate</a></li>
```

Comment:
Looks like the "Collaborate" target  changed, but I agree that the contributing page is a better target.